UKHomeOffice/cerberus-service
# COP-9367 New task API

## Description
To enable checkbox filters and filter counts we have a new API to use to get tasks & counts

- it refactors the getting counts to use the new API
- This PR **breaks** filtering temporarily

## To Test

- update your run script to remove `/engine-rest` from the camunda URL
- go to /tasks
- task counts in tabs should reflect the number of tasks for the status
- pagination should work (you can test it by setting `itemsPerPage` to a lower number on line 92)

# DO NOT MERGE THIS PR
It breaks filters and they are being recreated with the new API and will be merged into this branch before merging to dev

## Developer Checklist

\* Required
